spec/api-app-spec.js

```diff
@@ -805,33 +805,57 @@ describe('app module', () => {
   })
 
   describe('getGPUInfo() API', () => {
+    const appPath = path.join(__dirname, 'fixtures', 'api', 'gpu-info.js')
+
     it('succeeds with basic GPUInfo', (done) => {
-      app.getGPUInfo('basic').then((gpuInfo) => {
-        // Devices information is always present in the available info
-        expect(gpuInfo.gpuDevice).to.be.an('array')
-        expect(gpuInfo.gpuDevice.length).to.be.greaterThan(0)
+      const appProcess = ChildProcess.spawn(remote.process.execPath, [appPath, 'basic'])
+      let gpuInfoData = ''
+      appProcess.stdout.on('data', (data) => {
+        gpuInfoData += data
+      })
+      appProcess.stdout.on('end', () => {
```

Comment:
Maybe it would be better to make the test function `async` and write this line as 
`await emittedOnce(appProcess.stdout, 'end')` ?